imagecontent/registries.conf

```diff
@@ -9,17 +9,10 @@
 # and 'registries.block'.
 
 [registries.search]
-registries = ['docker.io', 'registry.fedoraproject.org', 'quay.io', 'registry.access.redhat.com', 'registry.centos.org']
+# registries = ['docker.io', 'registry.fedoraproject.org', 'quay.io', 'registry.access.redhat.com', 'registry.centos.org']
 
 # If you need to access insecure registries, add the registry's fully-qualified name.
 # An insecure registry is one that does not have a valid SSL certificate or only does HTTP.
 [registries.insecure]
 #registries = ['172.30.246.32:5000','172.30.1.1:5000','docker-registry.default.svc:5000','docker-registry.default.svc.cluster:5000','docker-registry.default.svc.cluster.local:5000']
 #registries = ['docker-registry.default.svc:5000','docker-registry.default.svc.cluster:5000','docker-registry.default.svc.cluster.local:5000','image-registry.openshift-image-registry.svc:5000','image-registry.openshift-image-registry.svc.cluster:5000','image-registry.openshift-image-registry.svc.cluster.local:5000']
-
-# If you need to block pull access from a registry, uncomment the section below
-# and add the registries fully-qualified name.
-#
-# Docker only
-[registries.block]
-registries = []
```

Comment:
i'd rather leave this (commented out) to at least provide a hint about the syntax.